### PR TITLE
Add sale transaction example

### PR DESCRIPTION
**Pull request by afiebig**
This PR adds sale transaction example in Console and Windows Form applications.

Also, refactor some method to comply with the .net coding standards.

### CODE REVIEW

**Review by a1b4 — APPROVED** [review]

File: TransbankPosSDKConsoleExample/Program.cs [hunk 1]
```diff
@@ -8,12 +9,29 @@ class Program
...
+            }
+            else
+            {
+                return;
+            }
 
-            pos.OpenPort(portName, TbkBaudrate.TBK_115200);
-            Console.WriteLine("Serial Port succesfuly opend");
+            POS.Instance.OpenPort(portName, TbkBaudrate.TBK_115200);
+            Console.WriteLine("\nSerial Port succesfuly opend");
```
Comment (leonelmunozlopez) [inline TransbankPosSDKConsoleExample/Program.cs]: type: "Serial port opened successfully"

**Review by leonelmunozlopez — CHANGES_REQUESTED** [review]
Just a typo

File: TransbankPosSDKExample/Form1.cs [hunk 2]
```diff
@@ -21,18 +22,17 @@ public Form1()
             port_ddown.DataSource = Serial.ListPorts();
             portName = port_ddown.SelectedItem.ToString();
             total_price_lbl.Text = total.ToString();
-
-            pos = new POS();
+            buyItems = new List<Product>();
         }
 
-        private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
+        private void ComboBox1_SelectedIndexChanged(object sender, EventArgs e)
```
Comment (leonelmunozlopez) [inline TransbankPosSDKExample/Form1.cs]: I know this just an example but.. we can use a descriptive Input name

File: TransbankPosSDKExample/Form1.cs [hunk 3]
```diff
@@ -42,38 +42,266 @@ private void pollingToolStripMenuItem_Click(object sender, EventArgs e)
...
+            {
+                Name = "Galletas",
+                Price = 1000
+            };
+
+            ListViewItem item = new ListViewItem(p.Row)
+            {
+                Tag = p
+            };
+            CarroCompraList.Items.Add(item);
```
Comment (leonelmunozlopez) [inline TransbankPosSDKExample/Form1.cs]: maybe: basket, cart or ShoppingCart

File: TransbankPosSDKExample/Form1.cs [hunk 4]
```diff
@@ -42,38 +42,266 @@ private void pollingToolStripMenuItem_Click(object sender, EventArgs e)
...
+
+            total += p.Price;
+            total_price_lbl.Text = total.ToString();
+            total_price_lbl.Refresh();
+        }
```
Comment (leonelmunozlopez) [inline TransbankPosSDKExample/Form1.cs:192]: Here we can use a common function with parameters or arguments like ```AddItem(name, qty)```

**Review by leonelmunozlopez — CHANGES_REQUESTED** [review]

**Pull request merged**